### ISSUE TITLE
Cannot access a homedir folder on my local machine

### ISSUE THREAD

**Issue by dzek69** [subhra74/snowflake#70]
**Describe the bug**
After connecting with default settings on the right panel (local files) I am starting in my homedir... but it seems empty. When I go up - everything works. But I cannot access my homedir. When I try - view reloads, path changes, but the files are stuck on `C:\Users`.

**To Reproduce**
1. Connect to any server on windows
2. Try to access your homedir

**Expected behavior**
Homedir should be accessible

**Screenshots**
See video: https://youtu.be/oT2TxWa4Iak

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: It doesn't matter, I use most of them
 - Version: App: 1.0.1, Windows: 10, 10.0.18362.592

**Additional context**
These are the files and folders that exist in my homedir (I think that this could be the source of the problem)
```
C:\Users\dzek>dir
 Volume in drive C has no label.
 Volume Serial Number is 525B-D0AA

 Directory of C:\Users\dzek

03.02.2020  15:53    <DIR>          .
03.02.2020  15:53    <DIR>          ..
14.01.2020  15:59    <DIR>          .android
14.01.2020  15:23    <DIR>          .AndroidStudio3.5
03.12.2019  20:28    <DIR>          .config
31.01.2020  14:18    <DIR>          .docker
03.12.2019  13:21    <DIR>          .eclipse
03.12.2019  14:15                81 .gitconfig
14.01.2020  15:53    <DIR>          .gradle
17.12.2019  15:19    <DIR>          .p2
24.01.2020  10:02                64 .pgAdmin4.2123502176.addr
24.01.2020  11:49             5 687 .pgAdmin4.2123502176.log
24.01.2020  10:02             1 577 .pgAdmin4.startup.log
03.12.2019  09:13    <DIR>          .PhpStorm2019.3
13.01.2020  16:43    <DIR>          .react-native-cli
03.12.2019  11:23    <DIR>          .ssh
03.02.2020  13:14               374 .vivaldi_reporting_data
19.01.2020  15:59               160 .yarnrc
29.01.2020  14:35    <DIR>          .zenmap
11.12.2019  10:39    <DIR>          3D Objects
29.01.2020  19:40                 0 abc.mp4
11.12.2019  10:39    <DIR>          Contacts
19.12.2019  13:12    <DIR>          Desktop
11.12.2019  10:39    <DIR>          Favorites
11.12.2019  10:39    <DIR>          Links
11.12.2019  10:39    <DIR>          Music
02.12.2019  19:08    <DIR>          node_modules
02.12.2019  13:45    <DIR>          OneDrive
02.12.2019  18:13    <DIR>          OpenVPN
11.12.2019  10:39    <DIR>          Saved Games
11.12.2019  10:39    <DIR>          Searches
03.02.2020  15:54    <DIR>          snowflake-ssh
09.12.2019  19:20    <DIR>          Temp
29.01.2020  19:38    <DIR>          Videos
02.12.2019  19:08                86 yarn.lock
               8 File(s)          8 029 bytes
              27 Dir(s)  118 125 735 936 bytes free

C:\Users\dzek>
```

Also I have moved original locations of my Desktop, Downloads, Documents folders (built-in Windows feature)

**Comment by IITFA** [comment]
I'm having this same issue, except I'm on Linux. MX 19 specifically.

**Comment by subhra74** [comment]
fixed in 1.0.4
